### ISSUE TITLE
Problems parsing date

### ISSUE THREAD

**Issue by victorleal** [fkammer/ng-eternicode-datepicker#3]
All of my dates are changed to one day before, for example: selecting 21/07/2016 on the datepicker, was setting the model (and the input) value to 20/07/2016.

After some research, looking inside your code, I saw that your directive use the method `getUTCDate` provided by bootstrap-datepicker.  

I didn't see the bootstrap-datepicker code responsible for getting the UTC date. But I've seen another issues regarding dates using the JavaScript Date object, **when it does actions with UTC**. Also, there is a bug opened for Chromium (https://bugs.chromium.org/p/chromium/issues/detail?id=589858) about this problem. 

It seems to be a more extended problem, so I don't know if there's a good workaround to this. Maybe we will have to wait.

**Comment by fkammer** [comment]
Hey @victorleal,

thanks for your message. Actually I though all that UTC stuff fixes that problem (I ran into it too).
You can read my post on the former git repo: https://github.com/cletourneau/angular-bootstrap-datepicker/issues/5#issuecomment-207871287

But thinking about it, this only works for timezones ahead of GMT (GMT + x). As you're behind GMT this the problem still occurs for you.

I'll look into that, try to fix it for all timezones in this repo and get in touch with eternicode to probably fix it in the main repository.

If you have any thoughts on this you're more than welcome to share.

**Comment by victorleal** [comment]
@fkammer Yes, this problem occurs specially for people who are behind GMT. I've said it on the Chromium thread also.

I think it will be hard to fix this problem, specially because Firefox has a different approach over this UTC parsing. 

Regarding a possible solution, setting the hour, minute and second to 00 worked for Chrome when using JavaScript Date object. We have to see if this workaround will work inside the directive or the datepicker.
